std/algorithm/searching.d

```diff
@@ -1792,6 +1796,7 @@ if (isRandomAccessRange!R1 && isBidirectionalRange!R2
     }
     // Stage 2: linear find
     size_t scout = needleLength - 1;
+outer:
```

Comment:
Is that consensus? There is one other label in the file "searching:" which does it differently.